dist/openapi/payment-initiation-openapi.yaml

```diff
@@ -3047,6 +3047,56 @@ components:
       type: string
       minLength: 1
       maxLength: 16
+    OBCategoryPurposeCode:
+      description: >-
+        Enumeration of codes that outlines the type of purpose behind a
+        transaction, payment or risk.
```

Comment:
No need for breaking the string over 2 lines

Reply:
This was generated from converting the relevant JSON, is it okay to leave this?